### PR TITLE
Add ability to create new users

### PR DESCRIPTION
**Pull request by erynplant**
- Add the ability to create new users
- Setup project to use Cloud SQL 
- Update README with new steps to run software locally

### CODE REVIEW

File: application/templates/application/documentation.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{% extends "application/global-html.html" %}
+{% block body %}
+    <div style="text-align: center;">
+        <div class="container mx-auto mt-4">
+            <div class="row">
+                <div class="col-md">
+                        <button class="btn btn-primary mt-4 btn-block"
+                                onclick="window.location.href={% url 'logout-view' %}">User Documentation
```
Comment (erynplant) [inline application/templates/application/documentation.html]: This should link to the github documentation page.
User: 'https://github.com/Brick7Face/psychiatric-guide-app/blob/master/User_Documentation.md'
Developer: 'https://github.com/Brick7Face/psychiatric-guide-app/blob/master/README.md'

File: application/templates/application/global-html.html [hunk 2]
```diff
@@ -13,18 +13,31 @@
...
-    <title>{{ title }}</title> <!-- creates title if available -->
+        <title>{{ title }}</title> <!-- creates title if available -->
     {% else %}
-    <title>Psychiatric Guide App</title> <!-- Generic Title -->
+        <title>Psychiatric Guide App</title> <!-- Generic Title -->
     {% endif %}
 </head>
 <header>
-    <center><h1>Application</h1></center>
+    <div style="text-align: center;"><h1>Application</h1></div>
```
Comment (erynplant) [inline application/templates/application/global-html.html:22]: We should do this styling in the css files. It looks like this is used a few times, so a css class called 'div-center' or something could be created. Then if we want to update the styling of this stuff, we will just have to change it in once place

File: application/templates/application/global-html.html [hunk 3]
```diff
@@ -40,22 +53,29 @@
...
+    <div style="text-align: center;">
+        <div class="container mx-auto mt-4">
+            {% block footer %} <!-- Anything in this block can be overridden -->
+                <div class="row">
                 <div class="col-md">
-                    <a href="../admin" style="color: #A9A9A9">Admin Login</a>
-                </div>
-                <div class="col-md">
-                    <a href="../admin/auth/user/add/" style="color: #A9A9A9">Add User</a>
+                    <a href="{% url 'documentation' %}" style="color: #A9A9A9">Documentation</a>
```
Comment (erynplant) [inline application/templates/application/global-html.html:61]: CSS class for this stuff too

File: application/templates/application/global-html.html (hunk 2 above)
Comment (MadisonFichtner) [reply application/templates/application/global-html.html:22]: I have this CSS done. Should get updated when we merge

File: application/templates/application/global-html.html (hunk 3 above)
Comment (MadisonFichtner) [reply application/templates/application/global-html.html:61]: This is also done in my branch.

**Pull request merged**